apps/script/codemod-es6-modules.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e # Exit on error
+
+if [[ $# -lt 1 ]]; then
+    echo "Usage: npm run codemod:es6-modules [PATH...]"
+    exit 0;
+fi
+
+echo "Converting require statements to es6 import statements in $1"
+echo "If this doesn't work, make sure you are using node>4"
```

Comment:
If this requires node > 4, can't we just check for it instead of leaving user to guess if that's why things didnt work for them